website/source/docs/commands/acl.html.md

```diff
@@ -94,9 +94,12 @@ Usage: consul acl <subcommand> [options] [args]
   For more examples, ask for subcommand help or view the documentation.
 
 Subcommands:
```

Comment:
kait: The capitalization in this needs to be revisited since its consistent I don't think we need to do anything immediately.

Reply:
Note: this is coming from the `consul acl` subcommand output, so changing it requires updating the text embedded in the binary.